po/ru.po

```diff
@@ -0,0 +1,353 @@
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:67
+msgid "There's now a status bar showing the URL of hovered links"
+msgstr "Теперь в строке состояния отображается URL наведенных ссылок"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:68
+msgid "Various code refactorings"
+msgstr "Различные улучшения кода"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:73
+msgid "Relax download folder permissions to fix downloads not starting"
+msgstr ""
+"Расслабление разрешений папки загрузки для устранения не запускающихся "
+"загрузок"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:79
+msgid ""
+"Improved design, with completely new pages for downloads, input requests, "
+"external links, errors"
+msgstr ""
+"Улучшенный дизайн, с совершенно новыми страницами для загрузок, запросов на "
+"ввод, внешних ссылок, ошибок"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:80
+msgid ""
+"Added more spacing between links to make them easier to click on small "
+"screens"
+msgstr ""
+"Добавлено большее расстояние между ссылками, чтобы их было легче нажимать на "
+"маленьких экранах"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:81
+msgid ""
+"Added zoom functionality, via shortcuts or directly from the popover menu"
+msgstr ""
+"Добавлена функция масштабирования, через комбинации клавиш или "
+"непосредственно из выпадающего меню"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:82
+msgid "Streaming button (alpha) for some audio/video file types"
+msgstr ""
+"Кнопка потоковой передачи (альфа) для некоторых типов аудио/видео файлов"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:88
+msgid "Changes"
+msgstr "Изменения"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:90
+msgid "Fix downloads by granting access to download folder"
+msgstr "Исправлены загрузки, предоставив доступ к папке загрузок"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:91
+msgid "Improve download progress handling"
+msgstr "Улучшена обработка хода загрузки"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:92
+msgid "Show close button on small window sizes"
+msgstr "Отображение кнопки закрытия окна небольшого размера"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:97
+msgid ""
+"This is the first working flatpak release. Some features (colors, caching) "
+"are missing."
+msgstr ""
+"Это первый рабочий релиз flatpak. Некоторые функции (цвета, кэширование) "
+"отсутствуют."
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:104
+msgid "First release on flathub! You get all the most important features"
+msgstr "Первый релиз на flathub! Вы получите все самые важные функции"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:106
+msgid "colors"
+msgstr "цвета"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:107
+msgid "caching"
+msgstr "кэширование"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:108
+msgid "search"
+msgstr "поиск"
+
+#: data/com.ranfdev.Geopard.metainfo.xml.in.in:139
+msgid "Lorenzo Miglietta"
+msgstr "Лоренцо Миглиэтта"
```

Comment:
Wut, really my name can be translated in Russian?

Reply:
Well, not really, it's just how it would sound in russian :smile: I can keep it untranslated

Reply:
Yeah I'd prefer we keep it untranslated